manifests/init.pp

```diff
@@ -42,6 +42,17 @@
 #   3 and the first will be Unbound on the local server,
 #   see http://man7.org/linux/man-pages/man5/resolv.conf.5.html
 #
+# @param disable_unbound_anchor
+#   Sets the unbound sysconfig startup setting 'DISABLE_UNBOUND_ANCHOR' to yes
+#   or no
+#
+#   Default for DISABLE_UNBOUND_ANCHOR in no, so you will only see this
+#   startup setting in /etc/sysconfig/unbound if you set it to true.
+#
+#   Note that in almost all cases you should leave this at the default of
+#   false. Setting it to true should generally only be necessary if your DNS
+#   servers do not support DNSSEC.
+#
```

Comment:
I think it would be good to add two lines to the comments:

1) Note under `disable_unbound_anchor` that says something like "See also enable_harden_dnssec_stripped"
2) Note under `enable_harden_dnssec_stripped` that says something like "See also disable_unbound_anchor"

Mainly since if we flip one we will probably want to flip the other.

Debated if we just collapse these two separate booleans into a single one like disable_DNSSEC, but sometime down the road it may be useful to control both those settings independently so it's fine to keep them separate

Reply:
That makes sense.